modules/oe_paragraphs_media/config/install/field.field.paragraph.oe_text_feature_media.field_oe_link.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_link
+    - paragraphs.paragraphs_type.oe_text_feature_media
+  module:
+    - link
+id: paragraph.oe_text_feature_media.field_oe_link
+field_name: field_oe_link
+entity_type: paragraph
+bundle: oe_text_feature_media
+label: Link
+description: ''
+required: false
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings:
+  link_type: 17
+  title: 1
```

Comment:
We should set the title required when the link is filled in.

Reply:
It isn't defined in the specification but for consistency with usage of Link field in other paragraphs it was set.